analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveIdenticalImplementations.CSharp9.cs

```diff
@@ -48,3 +49,18 @@ public void Method4()
         Console.WriteLine("Result: 0");
     }
 }
+
+public record SamplePositional(string Value)
```

Comment:
Can you please add also expression bodied method examples?

Reply:
Do you think it makes sense, since rule ignores functions having less that one statement?

Reply:
Anyway, added an example